.github/workflows/presubmit.yaml

```diff
@@ -44,4 +47,37 @@ jobs:
       - name: Run presubmit (GPU off)
         run: make presubmit
       - name: Run presubmit (GPU on)
-        run: GO_BUILD_TAGS=gpu make presubmit
\ No newline at end of file
+        run: GO_BUILD_TAGS=gpu make presubmit
+
+  test-components:
+    strategy:
+      matrix:
+        distro: [google-otel, otelopscol]
+        go: [1.24.2]
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+          lfs: true
+      - uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go }}
+          cache: true
+      - name: Test Components
+        run: make test-${{matrix.distro}}-components
```

Comment:
Looks like the target is not running https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/actions/runs/14735595188/job/41360745228 - probably need to do `make workspace` first so that `go list` will list modules

Reply:
Done in [73d858c](https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/292/commits/73d858ce936698ad1db41e43388df9594aa0ba9a)